yarn.lock

```diff
@@ -478,6 +478,28 @@
     handlebars "4.7.7"
     yargs "^15.1.0"
 
+"@aws-amplify/graphql-transformer-core@^1.4.0":
```

Comment:
This is coming in due to an unused dependency on this package within the cloudformation-provider package, I'm removing that from the CLI as part of this migration. This should go away after we round trip the release.